packages/babel-parser/src/parser/statement.js

```diff
@@ -2042,6 +2058,27 @@ export default class StatementParser extends ExpressionParser {
     node.specifiers.push(this.finishNode(specifier, type));
   }
 
+  maybeParseModuleAttributes() {
+    if (!this.eat(tt._with)) {
+      if (this.hasPlugin("moduleAttributes")) return [];
+      return null;
+    }
+    this.expectPlugin("moduleAttributes");
+    const attrs = [];
+    do {
+      const node = this.startNode();
+      node.key = this.parseIdentifier();
+      node.computed = false;
+      node.method = false;
+      node.shorthand = false;
+      this.expect(tt.colon);
+      node.value = this.parsePrimitiveValue();
```

Comment:
Noted. Thank you @littledan 👍

Reply:
I agree with @littledan 

That said, we didn't define the grammar yet and we might want to consider things like:

```
import u from "a" with lazy: true;
```

instead of a string literal?

Reply:
Yes, we might want to consider it. But I don't think this is worth complicating the Babel codebase at the moment.